pkg/process/checks/process.go

```diff
@@ -112,7 +116,7 @@ type ProcessCheck struct {
 func (p *ProcessCheck) Init(syscfg *SysProbeConfig, info *HostInfo, oneShot bool) error {
 	p.hostInfo = info
 	p.sysProbeConfig = syscfg
-	p.probe = newProcessProbe(p.config, procutil.WithPermission(syscfg.ProcessModuleEnabled))
+	p.probe = newProcessProbe(p.config, procutil.WithPermission(syscfg.ProcessModuleEnabled), procutil.WithIgnoreZombieProcesses(p.config.GetBool(configDisallowZombies)))
```

Comment:
Optional: long lines can sometimes get hard to parse, we could break this up with a param on each line